src/pages/shared/PostComponents/Comments.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { PaperPlaneOutline } from 'react-ionicons';
+import { getComments, postComment } from '../../../service/service.posts';
+
+export default function Comments({token, postId, userInfo, postUserId, peopleIFollow}){
+	const [text, setText] = useState('');
+	const [commentsList, setCommentsList] = useState([]);    
+
+	useEffect(()=>{
+		getComments(token, postId).then((res)=>{
+			setCommentsList(res.data.comments);
+		}).catch();
+	},[commentsList]);
+
+
+	function newComment(){
+		postComment(token, postId, text).then(()=>{
+			setText('');
+		}).catch();
+	}
+    
+	return(
+		<CommentsContainer>
+			{(commentsList.length) ? commentsList.map((comment)=>{
+				return (
+					<>
+						<CommentBox key={comment.id}>
+							<UserIcon src={comment.user.avatar}/>
+							<CommentInfo>
+								<Username>{comment.user.username}</Username>
+								<UserTag>
+									{(comment.user.id === postUserId) 
+										? '• post\'s author' 
+										: (peopleIFollow.includes(comment.user.id)
+											?'• following'
+											: ''
+										)
+									}
+								</UserTag>
+							</CommentInfo>
+							<CommentText>{comment.text}</CommentText>
+						</CommentBox>
+						<Separator />
+					</>
+				); 
+			}): ''}			
+			<CommentBox>
+				<UserIcon src={userInfo.userImg} />
+				<CommentInput 
+					placeholder="write a comment..." 
+					type='text'
+					onChange={(e)=>setText(e.target.value)}
+					value={text}
+				/>
+				<SendComment onClick={newComment} >
+					<PaperPlaneOutline
+						color={'#fff'} 
+						height="17px"
+						width="17px"
+					/>
+				</SendComment>
+			</CommentBox>
+		</CommentsContainer>
```

Comment:
Opa, acho que deu alguma coisa errada com a indentação aqui.  Acho também que valeria a pena extrair o código que está dentro do map para um outro componente.

Reply:
boa!

Reply:
bruna, vc acha que faz mais sentido deixar os estilos do styled components dentro do proprio componente ou extrair todos pra um arquivo separado? visualmente eu acho que fica mais agradável, mas pra editar acho que fica mais trabalhoso

Reply:
Se for pouca coisa acho que não tem problema deixar no componente.

Reply:
prontinho, separei o componente!